src/main/java/com/openshift3/client/model/build/ISTIBuildStrategy.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc. Distributed under license by Red Hat, Inc.
+ * All rights reserved. This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution, and is
+ * available at http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors: Red Hat, Inc.
+ ******************************************************************************/
+package com.openshift3.client.model.build;
+
+import java.util.List;
+
+import com.openshift3.client.images.DockerImageURI;
+import com.openshift3.client.model.EnviromentVariable;
+
+public interface ISTIBuildStrategy extends IBuildStrategy {
+	
+	/**
+	 * The Builder Image used to execute the build
+	 */
+	DockerImageURI getImage();
+	
+	String getScriptsLocation();
+	
+	List<EnviromentVariable> getEnvironmentVariables();
```

Comment:
not sure getting a list of environment variables is helpful. I would think that if you are interested in environment variables, you'd be interested in their value for a given key/name. Thus a map seems more helpful.